src/Sparrow/Collections/ConcurrentSet.cs

```diff
@@ -98,5 +99,21 @@ public override string ToString()
         {
             return Count.ToString("#,#", CultureInfo.InvariantCulture);
         }
+
+        public void UnionWith(IEnumerable<T> other)
```

Comment:
where do you use this?

Reply:
https://github.com/ravendb/ravendb/blob/ac9ffd513581d7f2b0b626ffb8a9519579c22154/test/SlowTests/Client/Subscriptions/ConcurrentSubscriptionsTests.cs#L1338